controllers/kyma_controller.go

```diff
@@ -125,7 +125,7 @@ func (r *KymaReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.
 func (r *KymaReconciler) reconcile(ctx context.Context, kyma *v1beta2.Kyma) (ctrl.Result, error) {
 	if r.SyncKymaEnabled(kyma) {
 		remoteClient := remote.NewClientWithConfig(r.Client, r.KcpRestConfig)
-		if err := remote.InitializeSyncContext(&ctx, kyma,
+		if err := remote.InitializeSyncContext(ctx, kyma,
```

Comment:
wow, @jakobmoellerdev, it's so glad to see you continue work on this project 😄 

This ctx pointer is actually needed, with this pointer reference, the ctx altered with additional syncContext can be used in further steps, otherwise, when it needs to use `SyncContextFromContext`, the ctx used there will not with syncContext, which leads to `ErrIsNoSyncContext`.

Reply:
I just changed it now so that the ineffectual assignment is gone. I believe this method is a bit strange but now at least the ctx switch is fine again

Reply:
btw is there any way I can look into the backlog you guys currently have? Would like to contribute from time to time still?

Reply:
ok, now you changed back before this function gets reverted https://github.com/kyma-project/lifecycle-manager/pull/837/files#diff-f3f736c5fa557c8d1171caedf277f1364262cc378e79bcd147ed2d3d29659b17L145

That ineffectual assignment is because the `ctx` initiated in the last step is not used, the scope of that `ctx` without pointer is limited to the function body. 

But then let's change back to return this context specifically so that with the least surprise.

I tried to invite you as a reader role into the backlog project, but seems not possible, the backlog is configured as "Only those with access to this project can view it." This setting is unfortunately configured by the project owner, but I'll clarify if it's possible to have some exceptions.

Reply:
Understood. Yeah I agree both options are kind of hacky, i think for me its simply a matter of dont have a pointer on interface. But you're right, the pointer in this instance would also work. I guess its just style, can also close the PR if you want.

Regarding the collaboration, are there any features you are willing to outsource and that external contributors can work on?

Reply:
Of course, regarding our current progress, some topics in my mind which worth your attention:

We are enabling watcher atm, it's working but not been introduced to PROD yet, I am a bit concerned about its performance, it's just a single embedded server and can't scale, atm, it's probably fine to handle those requests, but will be problem when the coming request increased, btw, we introduced metric to monitoring it.

Another thing is regarding this [issue](https://github.com/kyma-project/lifecycle-manager/issues/836), it's also very valuable to have your opinion, basically, it needs to change current reconciliation logic, but I don't want to introduce much special logic to handle this situation, it should be generic enough.

One more thing appreciate your knowledge is regarding that ocm library, recently, I did an upgrade for go mod in this [PR](https://github.com/kyma-project/cli/pull/1758/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6L34), as you can see, the ocm still not updated to the latest [0.3.0](https://github.com/open-component-model/ocm/releases/tag/v0.3.0), the reason is between 0.3.0-rc1 and 0.3.0, lots of breaking changes introduced, they completely rewrite some code, for example, to register repository, in [0.3.0](https://github.com/open-component-model/ocm/blob/v0.3.0/pkg/runtime/scheme.go#L440), it changes method signature to generic type, it also have other breaking changes introduced. I actually plan to have a concrete issue for this ocm upgrade, both in CLI and LM. If you can provide some insights for this upgrade or work on it, it will be great.

Reply:
I saw you already have a PR for the ocm upgrade 👍 
And thanks for the comments on the reconciliation issue, we can continue to discuss the issue there.
btw, I mentioned your PR within team on the daily(we are currently only half of the team size, @janmedrek is on vacation until 18.09), everyone is happy about you continuing to contribute to this project😄

Reply:
and which open source Kyma slack channel?

Reply:
> and which open source Kyma slack channel?

http://slack.kyma-project.io/ as referenced in https://github.com/kyma-project/community#slack